src/main/resources/com/synopsys/integration/jenkins/scan/extension/pipeline/SecurityScanStep/config.jelly

```diff
@@ -1,49 +1,50 @@
-    <f:entry field="scan_type" title="Scan Type">
-        <f:select/>
-    </f:entry>
-    <f:entry field="bridge_blackduck_url" title="Bridge Black Duck URL">
-        <f:textbox/>
-    </f:entry>
-    <f:entry field="bridge_blackduck_api_token" title="Bridge Black Duck Api Token">
-        <f:password/>
-    </f:entry>
-    <f:advanced>
-        <f:section title="Additional Bridge Black Duck Parameters">
-            <f:entry field="bridge_blackduck_install_directory" title="Bridge Black Duck Installation Path">
-                <f:textbox/>
-            </f:entry>
-            <f:entry field="bridge_blackduck_scan_full" title="Bridge Black Duck Full Scan">
-                <f:checkbox default="true"/>
-            </f:entry>
-            <f:entry field="bridge_blackduck_scan_failure_severities" title="Bridge Black Duck Scan Failure Severities">
-                <f:textbox/>
-            </f:entry>
-            <f:entry field="bridge_blackduck_automation_prcomment" title="Bridge Black Duck Automatic Pull Request Comment">
-                <f:checkbox/>
-            </f:entry>
-            <f:entry field="bridge_blackduck_automation_fixpr" title="Bridge Black Duck Automatic Fix Pull Request">
-                <f:checkbox/>
-            </f:entry>
-            <f:entry field="bitbucket_token" title="Bitbucket Token">
-                <f:password/>
-            </f:entry>
-        </f:section>
-        <f:section title="Additional Synopsys Bridge Parameters">
-            <f:entry field="bridge_download_url" title="Synopsys Bridge Download URL">
-                <f:textbox/>
-            </f:entry>
-            <f:entry field="bridge_download_version" title="Synopsys Bridge Download Version">
-                <f:textbox/>
-            </f:entry>
-            <f:entry field="synopsys_bridge_path" title="Synopsys Bridge Path">
-                <f:textbox/>
-            </f:entry>
-            <f:entry field="include_diagnostics" title="Include Diagnostics">
-                <f:checkbox/>
-            </f:entry>
-        </f:section>
-    </f:advanced>
+    <f:section title="Additional Black Duck Parameters">
+        <f:entry field="bridge_blackduck_scan_full" title="Bridge Black Duck Full Scan">
+            <f:checkbox default="true"/>
+        </f:entry>
+        <f:entry field="bridge_blackduck_scan_failure_severities" title="Bridge Black Duck Scan Failure Severities">
+            <f:textbox/>
+        </f:entry>
+        <f:entry field="bridge_blackduck_automation_prcomment" title="Bridge Black Duck Automatic Pull Request Comment">
+            <f:checkbox/>
+        </f:entry>
+        <f:entry field="bridge_blackduck_automation_fixpr" title="Bridge Black Duck Automatic Fix Pull Request">
+            <f:checkbox/>
+        </f:entry>
+    </f:section>
 
+    <f:section title="Additional Coverity Parameters">
+        <f:entry field="bridge_coverity_connect_project_name" title="Project name in Coverity">
```

Comment:
All titles should be similar to **BlackDuck** titles => _Bridge Coverity project name_

Also update the other titles in **Coverity** and **Polaris**